assets/src/js/menu_editor.js

```diff
@@ -60,15 +62,20 @@ const menuEditorRestrictions = () => {
    * title changes, adding, removing, changing level and reordering of items
    */
   const observer = new MutationObserver(() => checkNavMenu(getCurrentConf()));
-  const observerOpts = { subtree: true, childList: true, attributes: false, };
-  const enableObserver = () => { observer.observe(menuContent, observerOpts); };
-  const disableObserver = () => { observer.disconnect(); };
+  const observerOpts = {subtree: true, childList: true, attributes: false};
+  const enableObserver = () => {
+    observer.observe(menuContent, observerOpts);
+  };
+  const disableObserver = () => {
+    observer.disconnect();
+  };
 
   /**
    * Show a notice message to inform to editors to prevent
    * using long texts to menu items
    */
   const showInfoNoticeMessage = () => {
+    // eslint-disable-next-line no-shadow
```

Comment:
Here instead of the ESLint disable we can probably just remove the line, since `menuEditorFooter` is declared earlier